### PR TITLE
Init setup for frontend

### PR DESCRIPTION
**Pull request by MrHardcode**
The auto-generated gitignore file covers the necessary folders

### CODE REVIEW

**Comment by MrHardcode** [comment]
This PR is a part of the first user story #1

**Review by Runi-VN — APPROVED** [review]
I deleted the auto-generated README

Should the following not be deleted?:
- public/favicon.ico
- public/logo192.png
- public/logo512.png

(nitpicking)

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (MalteMagnussen) [inline public/index.html]:
```suggestion
      content="3. Semester Projekt - Hold Krykke"
```

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    />
+    <link rel="apple-touch-icon" href="logo192.png" />
```
Comment (MalteMagnussen) [inline public/index.html]:
```suggestion
```

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (MalteMagnussen) [inline public/index.html]:
```suggestion
    <title>Hold Krykke</title>
```

File: public/manifest.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    {
+      "src": "logo192.png",
+      "type": "image/png",
+      "sizes": "192x192"
+    },
+    {
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
```
Comment (MalteMagnussen) [inline public/manifest.json]: Delete this.

Comment (MalteMagnussen) [reply public/manifest.json]: Also delete the actual logo files.

**Review by MalteMagnussen — CHANGES_REQUESTED** [review]

File: public/manifest.json (hunk 4 above)
Comment (MalteMagnussen) [reply public/manifest.json]: Did it myself in commit 491d756

**Review by MalteMagnussen — APPROVED** [review]

**Pull request merged**